ocean_provider/validation/algo.py

```diff
@@ -276,11 +336,11 @@ def validate_formatted_algorithm_dict(algorithm_dict, algorithm_did):
         if not container.get(key):
             return (
                 False,
-                "algorithm `container` must specify values for all of entrypoint, image and checksum.",
+                "algorithm.container.missing_entrypoint_image_checksum",
             )
 
     if not container["checksum"].startswith("sha256:"):
-        return False, "container checksum must start with sha256:"
+        return False, "algorithm.container.checksum_prefix"
```

Comment:
In my opinion from the client side perspective it would make more sense and would be easier to deal with the error on the market for example if we have the error structured in a more consistent way like `resource.action`. like we have here for eg `algorithm.file_unavailable` where the left hand side is the resource and the right one is the action. And would be nice to split the resources as we split the action

So for this case i see the error as following:
```suggestion
        return False, "algorithm_container.checksum_prefix"
```